readme.md

```diff
@@ -70,9 +72,19 @@ Solution consists of 5 logically divided parts (projects):
 Developer should write code for a new module in .Core, .Data and .Web projects as usual in Visual Studio locally on host machine.
 Also developer should write tests for the new module in .Test project.
 
-## How to debug module
+## How to run Virto Commerce Platform Manager
+
+Docker bind Virto Commerce Platform Manager web app to 80 port. So if you have locally installed IIS server you should to stop it or set for local IIS other than 80 port.
```

Comment:
Docker would bind VC Platform Manager web app to port 80 by default. So, if you also have IIS server running locally, consider stopping it or changing the port in order to resolve port conflict.

@me: personally, I'm missing the option to change the port in docker. Could you add such instructions?

Reply:
fixed